Objects/dictobject.c

```diff
@@ -640,7 +656,7 @@ new_keys_object(PyInterpreterState *interp, uint8_t log2_size, bool unicode)
     else
 #endif
     {
-        dk = PyObject_Malloc(sizeof(PyDictKeysObject)
+        dk = PyMem_Malloc(sizeof(PyDictKeysObject)
```

Comment:
Maybe adjust the alignment of the next two lines (they were previously aligned with the `(`